pkg/syncer/resourcesync/controller.go

```diff
@@ -365,79 +330,163 @@ func (c *Controller) checkSSAR(ctx context.Context, gvr schema.GroupVersionResou
 	return sar.Status.Allowed, nil
 }
 
-// stopUnusedSyncerInformers stop syncers for gvrs not in requiredGVRs.
-func (c *Controller) stopUnusedSyncerInformers(ctx context.Context, requiredGVRs map[schema.GroupVersionResource]bool) {
+// removeUnusedGVRs stop syncers for gvrs not in requiredGVRs.
+func (c *Controller) removeUnusedGVRs(ctx context.Context, requiredGVRs map[schema.GroupVersionResource]bool) error {
```

Comment:
stopUnusedSyncers

Reply:
The name was but the comment was still obsolete. I finally changed the comment.